intellst-api/src/Repository/IdentifiedCaseRepository.php

```diff
@@ -27,38 +27,64 @@ public function removeCases()
 
         return $this->createQueryBuilder('IdentifiedCase')
             ->delete()
-            ->Where('IdentifiedCase.firstDate < :date')
+            ->where('IdentifiedCase.firstDate < :date')
             ->setParameter(':date', $date)
             ->getQuery()
             ->execute();
     }
 
-    // /**
-    //  * @return IdentifiedCase[] Returns an array of IdentifiedCase objects
-    //  */
-    /*
-    public function findByExampleField($value)
+    public function getNewIdentifiedCase(int $day, float $temperature)
     {
-        return $this->createQueryBuilder('i')
-            ->andWhere('i.exampleField = :val')
-            ->setParameter('val', $value)
-            ->orderBy('i.id', 'ASC')
-            ->setMaxResults(10)
+        $date = new \DateTime();
+        $date->modify("-" . $day . " day");
+
+        return $this->createQueryBuilder('IdentifiedCase')
+            ->where('IdentifiedCase.firstDate > :date')
+            ->setParameter(':date', $date)
+            ->andWhere('IdentifiedCase.temperature > :temperature')
+            ->setParameter(':temperature', $temperature)
             ->getQuery()
-            ->getResult()
-        ;
+            ->execute();
     }
-    */
 
-    /*
-    public function findOneBySomeField($value): ?IdentifiedCase
+    public function getReturnAttempts(int $day, float $temperature, string $uuid)
     {
-        return $this->createQueryBuilder('i')
-            ->andWhere('i.exampleField = :val')
-            ->setParameter('val', $value)
+        $date = new \DateTime();
+        $date->modify("-" . $day . " day");
+
+        return $this->createQueryBuilder('IdentifiedCase')
+            ->where('IdentifiedCase.firstDate > :date')
+            ->setParameter(':date', $date)
+            ->andWhere('IdentifiedCase.temperature > :temperature')
+            ->setParameter(':temperature', $temperature)
+            ->andWhere('IdentifiedCase.uuid = :uuid')
+            ->setParameter(':uuid', $uuid)
+            ->getQuery()
+            ->execute();
+    }
+
+    public function getReturnAttempt(string $uuid, $data)
+    {
+        return $this->createQueryBuilder('IdentifiedCase')
+            ->where('IdentifiedCase.firstDate > :date')
+            ->setParameter(':date', $data)
+            ->andWhere('IdentifiedCase.uuid = :uuid')
+            ->setParameter(':uuid', $uuid)
+            ->getQuery()
+            ->getResult();
+    }
+
+    public function entranceAllowed(string $uuid)
```

Comment:
name isn't good, but I can't suggest better name as I don't have any idea what should this method do, also missing return type